homeassistant/components/osoenergy/__init__.py

```diff
@@ -78,4 +82,9 @@ def __init__(self, osoenergy: OSOEnergy, osoenergy_device: _T) -> None:
         """Initialize the instance."""
         self.osoenergy = osoenergy
         self.device = osoenergy_device
-        self._attr_unique_id = osoenergy_device.device_id
+        self._attr_device_info = DeviceInfo(
```

Comment:
For a followup PR, would be nice if this class could move to `entity.py`

Reply:
I will do the change in the next PR.